### ISSUE TITLE
My Ideas page transitions

### ISSUE THREAD

**Issue by Watson-B** [ezl/dailyidea.com#809]
@ikosovec When a user goes to My Ideas and then navigates between any of the tabs (My Ideas, Saved Ideas, Settings) the full page reloads. However only the content/interactions below those tabs are actually changing.
![image](https://user-images.githubusercontent.com/65913664/102664328-5d583280-4137-11eb-86e8-d1eb0a9222c8.png)
![image](https://user-images.githubusercontent.com/65913664/102664339-63e6aa00-4137-11eb-9ef6-2b3c3dfbcf66.png)
![image](https://user-images.githubusercontent.com/65913664/102664354-6943f480-4137-11eb-896a-2b823bfbe9cb.png)

Can we update this so that the top section (highlighted in red) stays constant and doesn't reload and stays sticky to the top of the page if the user scrolls through their/saved ideas?
![image](https://user-images.githubusercontent.com/65913664/102664503-c475e700-4137-11eb-8016-504fa2cc5cd7.png)

**Comment by ikosovec** [comment]
@vedmant - this looks like a question for your expertise. :)

**Comment by vedmant** [comment]
@Watson-B 

> Can we update this so that the top section (highlighted in red) stays constant and doesn't reload and stays sticky to the top of the page if the user scrolls through their/saved ideas?

This is possible, I can disable global loading and create custom loader specifically for this pages. This will increase complexity however.
@ezl Let me know when and if I should do this task.

**Comment by ezl** [comment]
I see. @vedmant can we get an estimate on approximate implementation time?

I do agree that this is something we want to do, and we *will* do it at some point, but based on whether it's a 1 day or 5 day task, I may prefer to kick it down the road (i.e. if it's a day, lets just do it, but if it's more, let's wait).

**Comment by vedmant** [comment]
@ezl It's not a large task, maybe 1 hour if no issues encountered.

**Comment by ezl** [comment]
@vedmant ah -- lets just do it then.

**Comment by vedmant** [comment]
Done, but appeared to be more complex than I expected, had to add another custom plugin for this as nuxt just didn't have appropriate way to enable / disable global loader per route transition, so I moved loader state to Vuex.

**Comment by vedmant** [comment]
BTW, top part (user info and bio) is still reloaded on every page. To make this work better I'll need to move user data to Vuex store, also rewrite components to load data with fetch instead of asyncData

**Comment by ezl** [comment]
ok. will review it and after that we can decide if we wan to do anything different.

have fun skiing!

**Comment by ezl** [comment]
@Watson-B can you take a look at this and see how it feels to you now? (check on beta.dailyidea)

From what I can tell, it looks like it HAS been improved, but the top part still "flashes" once the page loads -- though it's quite fast. I do think the experience is a bit better -- for example, the page is snappier and you can see the loader on the bottom very quickly.

However, as @vedmant said, I do think in order to *really* make it work well, it will require some re-writing.

(A) how do you feel about the updated experience?
(B) in terms of the rewrite and getting a really smooth experience, would you be ok with delaying the next step of this improvement until a later date?

**Comment by Watson-B** [comment]
@ezl I agree with what you said. It's definitely an improvement over what we had before. The flashing is jarring, but doesn't break the experience, and so I don't think it's a super high priority and am fine with delaying.

**Comment by ikosovec** [comment]
I'm moving it back to icebox.
